CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.2.8
+
+<!-- generated comparing v5.2.7..master -->
+
+_Jan 10, 2022_
+
+A big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Added `Typography` and `Switch` to the new design system.
```

Comment:
Should we mention `Joy`? @danilo-leal

Reply:
Nah, I wouldn't, not yet. Not as a highlight.